content/en/Getting started/local.md

```diff
@@ -160,32 +143,31 @@ NAME                                                          REFERENCE
 horizontalpodautoscaler.autoscaling/hello-hello-k8s-service   Deployment/hello-hello-k8s-service   <unknown>/80%, <unknown>/80%   1         3         1          84m
 ```
 
-When you are done, you can delete the local kubernetes cluster by running:
-```bash
-opta destroy --local
-```
+You can also use Opta's [secret management](https://docs.opta.dev/tutorials/secrets/) to inject sensitive data into your applications.
 
+To uninstall your application from your local Kubernetes cluster run
+
+```
+opta destroy --local -c hello.yaml
+```
 #### Example 2: Deploy a service using a local docker image
 
 Now, let's build an application locally and deploy it.
 
 For this example, we will use our hello application (also available on [github](https://github.com/run-x/opta-examples/tree/main/hello-app)).
 1. Make sure that you have these files locally: 
     - `hello.yaml` The service Opta file. 
-    - `local.yaml` The environment Opta file. 
     - `app.py` The application code.
     - `Dockerfile` The docker file to build the application.
-1. Set `image: AUTO` in `opta.yaml` to use the local image.
-1. Make a change to the returned text in `app.py` to validate that the new image is used.
+2. Set `image: AUTO` in `hello.yaml` to use the local image.
+3. Make a change to the returned text in `app.py` to validate that the new image is used.
 
 {{< tabs tabTotal="4" tabID="2" tabName1="opta.yaml" tabName2="local.yaml" tabName3="app.py" tabName4="Dockerfile" >}}
```

Comment:
opta.yaml -> hello.yaml

Reply:
also remove `tabName2="local.yaml" ` from this and renumber the names

Reply:
Fixed